lrpar/examples/calc_actions/build.rs

```diff
@@ -44,6 +44,7 @@ fn main() -> Result<(), Box<std::error::Error>> {
     // enough to fit all IDs in) as well as the input file (which must end in ".y" for lrpar, and
     // ".l" for lrlex).
     let lex_rule_ids_map = CTParserBuilder::<u8>::new()
+        .error_on_conflicts(false)
```

Comment:
There are no conflicts here (I think!), so we don't want to set this to `false` do we?

Reply:
Yes, I thought I'll leave it in as an example, but I'm happy to remove it.

Reply:
I want to encourage people not to do `error_on_conflicts(false)` unless they know they have conflicts (e.g. the Lua grammar), so I think we should remove this.